.baseline/checkstyle/checkstyle.xml

```diff
@@ -409,11 +409,9 @@
         <module name="JavadocMethod"> <!-- Java Style Guide: Where Javadoc is used -->
             <property name="scope" value="public"/>
             <property name="allowMissingParamTags" value="true"/>
-            <property name="allowMissingThrowsTags" value="true"/>
             <property name="allowMissingReturnTag" value="true"/>
-            <property name="minLineCount" value="99999999"/>
             <property name="allowedAnnotations" value="Override, Test"/>
-            <property name="allowThrowsTagsForSubclasses" value="true"/>
+            <property name="validateThrows" value="false"/>
```

Comment:
This seems more general than "allowThrowsTagsForSubclasses"?

Reply:
It's a bit of a complicated story

https://github.com/checkstyle/checkstyle/issues/7329

Long story short, the checks were basically broken before they were removed and won't work. We actually don't need to set "validateThrows"  to false either since it is false by default. I just wanted to make sure we were aware that we weren't validating any throw related tags.